### ISSUE TITLE
Events not sent to GA, parameters appends together in wrong format

### ISSUE THREAD

**Issue by iranagame** [nuxt-community/web-vitals-module#25]
Events not being sent to GoogleAnalytic provider!

When you inspect the request, you will see the URL in the following format:
`https://www.google-analytics.com/collect?v=1&t=eventtid=UA-XXXXXXXXX-Xcid=0.883871679636306.0.12895864114226074ec=Web%20Vitalsea=CLSel=v1-1628200888064-2067831876287dl=%2Fdh=https%3A%2F%2Fxyz.com%2Fni=truez=1628200898652`

As you can see, the request is invalid since the parameters doesn't implode with `&` symbol, the following format is  expected which is worked:
`https://www.google-analytics.com/collect?v=1&t=event&tid=UA-XXXXXXXXX-X&cid=0.883871679636306.0.12895864114226074&ec=Web%20Vitals&ea=CLS&el=v1-1628200888064-2067831876287&dl=%2F&dh=https%3A%2F%2Fxyz.com%2F&ni=true&z=1628200898652`

**Comment by iranagame** [comment]
`encodeParams(...)`  in `src/runtime/utils.ts ` must be corrected as follows to fix the issue:

```
export function encodeParams (obj) {
  let str = ''
  for (let k in obj) {
    if (obj[k]) {
      str += `&${k}=${encodeURIComponent(obj[k])}`
    }
  }
  return str
}
```

**Comment by iranagame** [comment]
@pi0 Please check & merge ASAP, as module not working properly.

**Comment by Aliaaaam** [comment]
Sorry it's bit off-topic but since this PR has been merged, how can we add the main repo into our project through NPM?

**Comment by pi0** [comment]
Released in v0.1.7.
